### DISCUSSION TITLE
Token Joiner

### DISCUSSION THREAD

**Discussion by toastyy00** [rigwild/discord-self-bot-console#54] [Q&A]
is it possible to make token joiner?So join the server with multiple tokens

**Comment by DeadLyBro** [comment]
I can't find Join Server on Api. That is may cause Guild/Server Raid so it's not good bro. I don't think Discord may support this. 
or if there is code for other languages you can try. But i don't recommend.

**Reply by DeadLyBro** [reply]
or if u say using code with multiple tokens, you asked and Rigwild make a reply. 
Rigwild's Comment: https://github.com/rigwild/discord-self-bot-console/discussions/21#discussioncomment-3048097
